whatis277/harvest
# bean: Add subscriptions controller

Moves subscriptions create and delete handlers to subscriptions controller

Uses new paths for creating subs and deleting subs

Cleans up handlers

Testing instructions:
1. `dc up --build bean`
2. Ensure there's no errors in the logs
3. `dc exec -e INTEGRATION=1 bean go test ./...`
4. Ensure the tests pass
5. Visit http://localhost:8080/
6. Attempt to add subscriptions 
7. Attempt to delete the subscriptions you added
8. It should work properly
